src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,50 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
 }
 
+Animal.alive = [];
+
+Animal.checkIsAlive = () => {
+  for (const animal of Animal.alive) {
+    if (animal.health === 0) {
+      Animal.alive.splice(Animal.alive.indexOf(animal), 1);
+    }
+  }
+};
+
```

Comment:
You dont need to add new method in animal,you can check for alive animals in method bite use filter (checklist.2)

Reply:
agree, you can use filter